### ISSUE TITLE
The detection result by WebGL2 backend is different from the one by WASM backend on Android Pixel 2.

### ISSUE THREAD

**Issue by BruceDai** [intel/webml-polyfill#182]
<img src="https://user-images.githubusercontent.com/1531742/45034022-dc212280-b088-11e8-98f0-226d111c408b.png" height="480" width="270">
<img src="https://user-images.githubusercontent.com/1531742/45034039-e4795d80-b088-11e8-8e3b-d092a121cf88.png" height="480" width="270">

**Comment by BruceDai** [comment]
This issue is not reproduced on Mac OS platform. @sunlin-link PTAL

**Comment by BruceDai** [comment]
This issue is also reproduced by Chrome (68.0.3440.91), so it's relevant to the changed code of webgl, we will bisect the commit.

**Comment by BruceDai** [comment]
console log says:
webml-polyfill.js:955 Uncaught (in promise) Error: [Tensor] numSlices 22 > MAX_TEXTURE_IMAGE_UNITS 16
    at Tensor._createTextureSlices (webml-polyfill.js:955)
    at Tensor.createGLTexture (webml-polyfill.js:888)
    at Input.call (webml-polyfill.js:21574)
    at webml-polyfill.js:8169
    at Map.forEach (<anonymous>)
    at webml-polyfill.js:8168
    at Array.forEach (<anonymous>)
    at webml-polyfill.js:8158
    at new Promise (<anonymous>)
    at Model._execute (webml-polyfill.js:8150)

**Comment by BruceDai** [comment]
The root case is that in https://github.com/intel/webml-polyfill/blob/master/src/nn/webgl2/Tensor.js#L119
```javascript
    L119 const numSlices = Math.ceil(this.textureShape[0] / webgl2.MAX_TEXTURE_SIZE);
    L120 if (numSlices > webgl2.MAX_TEXTURE_IMAGE_UNITS) {
    L121   throw new Error(`[Tensor] numSlices ${numSlices} > MAX_TEXTURE_IMAGE_UNITS ${webgl2.MAX_TEXTURE_IMAGE_UNITS}`);
    L122 }
```
On Android platform **webgl2.MAX_TEXTURE_SIZE** is 4096 
while on Windows and Mac OS platforms **webgl2.MAX_TEXTURE_SIZE**  both are 16384

So execution on Android platform, the #L120 if condition would be satisfied, then error would be thrown.

**Comment by GreyZzzzzzXh** [comment]
Hi @BruceDai , I remove this check code via #215 , could you please help verify whether the result is correct?

**Comment by GreyZzzzzzXh** [comment]
Fixed by #215.

**Comment by Christywl** [comment]
Verified this with the latest codes, it can't reproduce.
